docs/OV_Runtime_UG/ONNX_Support.md

```diff
@@ -0,0 +1,65 @@
+# ONNX Format Support {#ONNX_Format_Support}
+
+
+Since the 2020.4 release, OpenVINO™ has supported native usage of ONNX models. The `core.read_model()` method provides a uniform way to read OpenVINO IR and ONNX formats alike, which is the recommended approach to reading models. Example:
```

Comment:
I suggest changing this to:

"Since the 2020.4 release, OpenVINO™ has supported native usage of ONNX models. The `core.read_model()` method, which is the recommended approach to reading models, provides a uniform way to read OpenVINO IR and ONNX formats alike. Example:"

This helps make it more clear that `core.read_model()` is the recommended way to read models.

Reply:
done :)